src/client/datascience/commands/activeEditorContext.ts

```diff
@@ -70,10 +69,6 @@ export class ActiveEditorContextService implements IExtensionSingleActivationSer
             EditorContexts.IsInteractiveOrNativeActive,
             this.commandManager
         );
-        this.canRunCellsAboveInNativeNotebook = new ContextKey(
-            EditorContexts.canRunCellsAboveInNativeNotebook,
```

Comment:
This context doesn't serve a purpose anymore. Since you can click the run above/below buttons on a cell, the selected cell doesn't matter.

This made it so run above was disabled on all cells, until you selected a cell that's not the first one.

I looked into it, but there's no way of having a context key at the cell level to just disable runAbove on the first cell. The only consequence is that runAbove is available on the first cell and clicking it does nothing.

We could ask vscode to create a context for the cell index. Let me know what you think.